pkgs/tools/inputmethods/fcitx5/with-addons.nix

```diff
@@ -38,10 +38,13 @@ symlinkJoin {
 
     desktop=share/applications/org.fcitx.Fcitx5.desktop
     autostart=etc/xdg/autostart/org.fcitx.Fcitx5.desktop
+    dbus=share/dbus-1/services/org.fcitx.Fcitx5.service
     rm $out/$desktop
     rm $out/$autostart
+    rm $out/$dbus
     cp ${fcitx5}/$desktop $out/$desktop
-    sed -i $out/$desktop -e "s|^Exec=.*|Exec=$out/bin/fcitx5|g"
+    cp ${fcitx5}/$dbus $out/$dbus
+    substituteInPlace $out/$desktop $out/$dbus --replace-fail ${fcitx5} $out
     ln -s $out/$desktop $out/$autostart
   '';
```

Comment:
Would you like to update other desktop files to addons one as well, though they are the same thing now?

Reply:
Absolutely. Updated.